packages/graphql/src/modules/adoption/AdoptionQuery.ts

```diff
@@ -0,0 +1,15 @@
+import {GraphQLFieldConfigMap, GraphQLList} from 'graphql';
+
+import {GraphQLContext} from '../../types';
+
+import AdoptionType from './AdoptionType';
+import AdoptionModel from './AdoptionModel';
+
+export const AdoptionsQueries: GraphQLFieldConfigMap<any, GraphQLContext, any> = {
+  adoptions: {
+    type: GraphQLList(AdoptionType),
```

Comment:
Shouldn’t this be a connection? https://relay.dev/docs/en/graphql-server-specification#connections

How many adoptions this query would return? If is more than 10, it should be a connection. If is not, then leave as it is.